<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-1-g15f4949  Build Version: 1.6.7594.29634
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="robotarmhackathon">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="104.20674769568 2.14038854680129 48.9094117884757"
        rpy="0 0 0" />
      <mass
        value="10467613.2406238" />
      <inertia
        ixx="13107519632.3634"
        ixy="-55188521.1955575"
        ixz="-1605823.10235128"
        iyy="23918500476.9016"
        iyz="109816.326054326"
        izz="10845868482.5405" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="base_leg_1">
    <inertial>
      <origin
        xyz="99.9315149733181 -85.7872417456713 -44.1134507472189"
        rpy="0 0 0" />
      <mass
        value="14024454.6532922" />
      <inertia
        ixx="2322712012.02237"
        ixy="-6153470.93379884"
        ixz="-9980267.40975024"
        iyy="3164204483.44981"
        iyz="-932011049.07348"
        izz="2260407763.54338" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/base_leg_1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/base_leg_1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="base_leg_1"
    type="fixed">
    <origin
      xyz="0.0331750659552412 0.035150719337271 67.6999969482422"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="base_leg_1" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="base_leg_2">
    <inertial>
      <origin
        xyz="99.9315149733181 -85.7872417456714 -44.1134507472189"
        rpy="0 0 0" />
      <mass
        value="14024454.6532921" />
      <inertia
        ixx="2322712012.02236"
        ixy="-6153470.93379897"
        ixz="-9980267.40975122"
        iyy="3164204483.4498"
        iyz="-932011049.073476"
        izz="2260407763.54337" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/base_leg_2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/base_leg_2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="base_leg_2"
    type="fixed">
    <origin
      xyz="220.986813987056 -149.227297525002 -22.7000007629394"
      rpy="3.14159265358979 0 1.5707963267949" />
    <parent
      link="base_link" />
    <child
      link="base_leg_2" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="base_leg_3">
    <inertial>
      <origin
        xyz="99.9315149733181 -85.7872417456713 -44.1134507472189"
        rpy="0 0 0" />
      <mass
        value="14024454.6532921" />
      <inertia
        ixx="2322712012.02237"
        ixy="-6153470.93379867"
        ixz="-9980267.40975017"
        iyy="3164204483.44981"
        iyz="-932011049.073479"
        izz="2260407763.54338" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/base_leg_3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/base_leg_3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="base_leg_3"
    type="fixed">
    <origin
      xyz="200.053083215943 -98.2058725159484 67.6999969482422"
      rpy="0 0 3.14159265358979" />
    <parent
      link="base_link" />
    <child
      link="base_leg_3" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="axis_1_base">
    <inertial>
      <origin
        xyz="95.9375121939249 -380.630664102495 -21.8762551208198"
        rpy="0 0 0" />
      <mass
        value="40932281.1699308" />
      <inertia
        ixx="14369922660.4427"
        ixy="-2077351721.99651"
        ixz="-399238146.758099"
        iyy="20887549884.6441"
        iyz="234293608.920345"
        izz="33961580084.1404" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/axis_1_base.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/axis_1_base.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="axis_1_base"
    type="fixed">
    <origin
      xyz="-3.6672 330.04 67.7"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="axis_1_base" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="axis_1_child">
    <inertial>
      <origin
        xyz="-0.57045784518418 -0.847850206558562 25.4069730097391"
        rpy="0 0 0" />
      <mass
        value="42138053.5896542" />
      <inertia
        ixx="44674977820.4956"
        ixy="-1139948725.46951"
        ixz="3663987083.6928"
        iyy="36428923022.2857"
        iyz="2198460579.50893"
        izz="27528388109.4216" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/axis_1_child.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/axis_1_child.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="axis_1_child"
    type="continuous">
    <origin
      xyz="99.999 -379.22 -0.05"
      rpy="0 0 0" />
    <parent
      link="axis_1_base" />
    <child
      link="axis_1_child" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="arm1">
    <inertial>
      <origin
        xyz="-37.4809094503138 0.801315982716993 34.5576406838505"
        rpy="0 0 0" />
      <mass
        value="33427957.8252588" />
      <inertia
        ixx="28352419324.3511"
        ixy="701152712.603097"
        ixz="27013648567.803"
        iyy="56973636333.2576"
        iyz="-645733310.894033"
        izz="32876966176.0036" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/arm1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/arm1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="arm1"
    type="continuous">
    <origin
      xyz="0.033422 0 61.071"
      rpy="0 0.82644 3.1416" />
    <parent
      link="axis_1_child" />
    <child
      link="arm1" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="arm2">
    <inertial>
      <origin
        xyz="66.3717579477605 -5.6523979593916 33.0767881043029"
        rpy="0 0 0" />
      <mass
        value="36928888.8607213" />
      <inertia
        ixx="14362604458.5882"
        ixy="-1227795195.20732"
        ixz="-21476693883.7102"
        iyy="57955982621.5579"
        iyz="23540789.3879213"
        izz="48768974871.0532" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/arm2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/arm2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="arm2"
    type="continuous">
    <origin
      xyz="-88.064 0 81.116"
      rpy="0 1.2805 3.1416" />
    <parent
      link="arm1" />
    <child
      link="arm2" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="endeffector1">
    <inertial>
      <origin
        xyz="133.627013638223 -54.1219380079637 199.011710293106"
        rpy="0 0 0" />
      <mass
        value="12888695.7038069" />
      <inertia
        ixx="2882385634.16868"
        ixy="-232380366.832862"
        ixz="253811824.524402"
        iyy="1817042019.06422"
        iyz="553056013.483461"
        izz="3244707577.17995" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/endeffector1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/endeffector1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="endeffector1"
    type="continuous">
    <origin
      xyz="86.484 45.917 -176.24"
      rpy="0 -0.45405 0" />
    <parent
      link="arm2" />
    <child
      link="endeffector1" />
    <axis
      xyz="0 -1 0" />
  </joint>
  <link
    name="endeffector2">
    <inertial>
      <origin
        xyz="-0.00395911319455422 3.23692438356659 4.85981240420523"
        rpy="0 0 0" />
      <mass
        value="22357667.7030269" />
      <inertia
        ixx="7153844560.42696"
        ixy="3276759.70488864"
        ixz="462212.01745629"
        iyy="5769081326.00476"
        iyz="-277445932.301696"
        izz="12310122424.0219" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/endeffector2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robotarmhackathon/meshes/endeffector2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="endeffector2"
    type="fixed">
    <origin
      xyz="164.540512084961 -49.5 205.348281860352"
      rpy="3.14159265358979 0 1.5707963267949" />
    <parent
      link="endeffector1" />
    <child
      link="endeffector2" />
    <axis
      xyz="0 0 0" />
  </joint>
</robot>